tide_landing_page.install

```diff
@@ -517,3 +518,17 @@ function tide_landing_page_update_8014() {
     }
   }
 }
+
+/**
+ * Adds youtube option paragraph type to field_paragraph_social_list.
+ */
+function tide_landing_page_update_8015() {
+  $field_storage = FieldStorageConfig::loadByName('paragraph', 'field_paragraph_social_list');
+  $new_field_storage = $field_storage->toArray();
+  $new_field_storage['settings']['allowed_values']['youtube_channel'] = 'Youtube channel';
+  $new_field_storage = FieldStorageConfig::create($new_field_storage);
```

Comment:
why creating a new field config?

Reply:
@MdNadimHossain , you might want to do $field_storage->setSettings(......)->save();

Reply:
Hi @vincent-gao 
Just saving it didn't work when updating the storage config, so followed the instructions from here.
https://www.drupal.org/docs/8/api/update-api/updating-entities-and-fields-in-drupal-8#s-updating-field-storage-config-items

Reply:
Hi @MdNadimHossain ,the instruction is for changing field type.but for changing the value.
if you do push your `youtube_channel` to `allowed_values`. and `$field_storage-> setSetting('allowed_values', $new_field_storage['settings']['allowed_values'])->save()` it should work. I think

Reply:
Ok, I will try to build locally like this again, if it works I will update this. thx

Reply:
@vincent-gao  I have made the changes. I was converting it to toArray() first, that is why that way wasn't working. But now directly did it without converting it.